skimage/color/colorconv.py

```diff
@@ -1516,10 +1526,12 @@ def combine_stains(stains, conv_matrix):
     from ..exposure import rescale_intensity
 
     stains = _prepare_colorarray(stains)
-    logrgb2 = -np.reshape(stains, (-1, 3)) @ conv_matrix
-    rgb2 = np.power(10, logrgb2)
-    return rescale_intensity(np.reshape(rgb2 - 2, stains.shape),
-                             in_range=(-1, 1))
```

Comment:
Is this change of behavior going to cause problems.

Reply:
It's not supposed to. I made some tests, and it works. Let's hope for the best :slightly_smiling_face: